JASP-Engine/JASP/R/correlationbayesian.R

```diff
@@ -239,9 +239,11 @@ CorrelationBayesian <- function(dataset=NULL, options, perform="run",
 	
 	if (flagSupported) {
 		if (bayesFactorType=="LogBF10") {
-			.addFootnote(footnotes, paste(bfTitle, " > log(10), ** , ", bfTitle, " > log(30), *** ", bfTitle, " > log(100)"), symbol="*")
+			.addFootnote(footnotes, paste(bfTitle, " > log(10), ** ", bfTitle, " > log(30), *** ", bfTitle, " > log(100)"), symbol="*")
+		} else if (bayesFactorType == "BF01") {
+			.addFootnote(footnotes, paste(bfTitle, " < .1, ** ", bfTitle, " < .03, *** ", bfTitle, " < .01"), symbol="*")
```

Comment:
.addFootnote(footnotes, paste(bfTitle, " < 10, ** ", bfTitle, " < 30, *** ", bfTitle, " < 100"), symbol="*")

Reply:
We're looking for evidence for model "1" though.
Let's say:
marginal model 0: 100
marginal model 1: 500

BF01 = 0.2
BF10 = 5

As marginal of 1 increases we gain more evidence and so if BF01 < 0.1 add a star, or if BF10 > 10 add a star. Right?